qrenderdoc/Windows/BufferViewer.cpp

```diff
@@ -4632,8 +4632,9 @@ void BufferViewer::on_outputTabs_currentChanged(int index)
   ui->autofitCamera->setEnabled(!isCurrentRasterOut());
 
   EnableCameraGuessControls();
-  ui->axisMappingCombo->setEnabled(index != 1);
-  ui->axisMappingButton->setEnabled(index != 1 && ui->axisMappingCombo->currentIndex() == 4);
+  ui->axisMappingCombo->setEnabled(!isCurrentRasterOut());
```

Comment:
I noticed when testing with a capture using tessellation that these controls seemed to be enabled and disabled for the wrong tabs. This should fix it so the controls are disabled for the output where we are unprojecting the vertices.

Reply:
Good catch, thanks!